structs/config.go

```diff
@@ -23,6 +23,7 @@ type Server struct {
 type PageContext struct {
 	LongName  string `toml:"longName"`
 	ShortName string `toml:"shortName"`
+	UrlPrefix string `toml:"urlPrefix"`
```

Comment:
vscode keeps complaining that we don't use URL instead or Url might be nicer to use URL everywhere